content/news/2017/08/05/release-1.3.2.html

```diff
@@ -139,7 +139,8 @@
       <h1>Apache Flink 1.3.2 Released</h1>
 
       <article>
-        <p>05 Aug 2017</p>
+        <p>05 Aug 2017
```

Comment:
would be neat if you could revert these; the next regeneration may reverse them again.